project/Dependencies.scala

```diff
@@ -2,7 +2,7 @@ import sbt._
 import scala.language.reflectiveCalls
 object WellcomeDependencies {
 
-  val defaultVersion = "32.41.1" // This is automatically bumped by the scala-libs release process, do not edit this line manually
+  val defaultVersion = "32.42.0" // This is automatically bumped by the scala-libs release process, do not edit this line manually
     "32.40.4" // This is automatically bumped by the scala-libs release process, do not edit this line manually
```

Comment:
I don't think it's a problem because it's been this way for a few commits on main but it's worth removing this duplicate line